### ISSUE TITLE
Aegis 2FA giving me wring codes only on my chromebook

### ISSUE THREAD

**Issue by kram63** [beemdevelopment/Aegis#937]
I notice today that Aegis gives me incorrect codes on my Chromebook.

I have aegis 2fa on my Chromebook & my android device. It gives me the correct codes on my android device but not my Chromebook.

In the past it's worked without a problem And I also have the keys so what I did is I deleted every entry and manually added a few, and tested them and Aegis gave me the wrong codes.

I know the keys are correct because I tested them with The other two-step authentication that I use (2FAS) and they match.

 I  doubt it is a  time issue with my Chromebook because 2FAS works no issues and 

I know the keys I have are correct I checked them with my 2FAS keys.

 it only is incorrect on the Chromebook.

Aegis works 100% on my Samsung mobile pH so not a huge issue BUT curious why I get wrong codes now on my Chromebook?

**Comment by alexbakker** [comment]
Indeed, this is almost certainly a system time issue. Perhaps 2FAS works because it consults the internet to obtain the current time.
